parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -256,6 +256,18 @@ try
 
         Write-KubeClusterConfig -MasterIP $MasterIP -KubeDnsServiceIp $KubeDnsServiceIp
 
+        Write-Log "Download kubelet binaries and unzip"
+        Get-KubePackage -KubeBinariesSASURL $global:KubeBinariesPackageSASURL
+
+        # this overwrite the binaries that are download from the custom packge with binaries

```

Comment:
I think the first line is not needed in the comment section (i know you are just moving it around but wouldn't hurt to clean it up)